commands/env.cmd

```diff
@@ -29,6 +29,12 @@ if [[ ${WARDEN_ENV_TYPE} != local ]]; then
     WARDEN_REDIS=${WARDEN_REDIS:-1}
 fi
 
+if [[ ${WARDEN_ENV_TYPE} == "magento1" && -f "${WARDEN_ENV_PATH}/.modman/.basedir" ]]; then
+  export WARDEN_MODMAN_ROOT='/'`cat "${WARDEN_ENV_PATH}/.modman/.basedir"`
```

Comment:
I am not a bash expert at all, but https://www.shellcheck.net/ recommends to [not use backticks](https://github.com/koalaman/shellcheck/wiki/SC2006). `'/'$(cat "${WARDEN_ENV_PATH}/.modman/.basedir")` should be a proper alternative.

Reply:
Thanks, adpated